presto-geospatial/src/test/resources/us-states.tsv

```diff
@@ -35,7 +35,7 @@ ND	POLYGON ((-97.228743 49.000239, -97.097296 48.682577, -97.16302 48.545653, -9
 OH	POLYGON ((-80.518598 41.978802, -80.518598 40.636951, -80.666475 40.582182, -80.595275 40.472643, -80.600752 40.319289, -80.737675 40.078303, -80.830783 39.711348, -81.219646 39.388209, -81.345616 39.344393, -81.455155 39.410117, -81.57017 39.267716, -81.685186 39.273193, -81.811156 39.0815, -81.783771 38.966484, -81.887833 38.873376, -82.03571 39.026731, -82.221926 38.785745, -82.172634 38.632391, -82.293127 38.577622, -82.331465 38.446175, -82.594358 38.424267, -82.731282 38.561191, -82.846298 38.588575, -82.890113 38.758361, -83.032514 38.725499, -83.142052 38.626914, -83.519961 38.703591, -83.678792 38.632391, -83.903347 38.769315, -84.215533 38.807653, -84.231963 38.895284, -84.43461 39.103408, -84.817996 39.103408, -84.801565 40.500028, -84.807042 41.694001, -83.454238 41.732339, -83.065375 41.595416, -82.933929 41.513262, -82.835344 41.589939, -82.616266 41.431108, -82.479343 41.381815, -82.013803 41.513262, -81.739956 41.485877, -81.444201 41.672093, -81.011523 41.852832, -80.518598 41.978802, -80.518598 41.978802))
```

Comment:
OH is also buggy, first, previous to last, and last are the same point.

Reply:
Would you fix OH polygon?

Reply:
done